packages/runtime-dom/src/modules/style.ts

```diff
@@ -2,20 +2,26 @@ import { isString } from '@vue/shared'
 
 export function patchStyle(el: any, prev: any, next: any) {
   const { style } = el
+
   if (!next) {
     el.removeAttribute('style')
-  } else if (isString(next)) {
+    return
+  }
+
+  if (isString(next)) {
     style.cssText = next
-  } else {
-    for (const key in next) {
-      style[key] = next[key]
-    }
-    if (prev && !isString(prev)) {
-      for (const key in prev) {
-        if (!next[key]) {
-          style[key] = ''
-        }
-      }
+    return
+  }
+
+  Object.assign(style, next)
```

Comment:
Is this `Object.assign` good? I can do a seperate PR.

/cc @yyx990803

Reply:
It's also noticeably slower than `for...in`.